README.md

```diff
@@ -167,21 +190,43 @@ Sends the given `message` to the `toPhoneNumber` from the `fromPhoneNumber`.
 
 **Returns:** A promise that is resolved when the message is sent.
 
+### sendAlertSessionUpdate(sessionId, responderPushId, toPhoneNumber, fromPhoneNumber, textMessage, pushMessage)
```

Comment:
is this function fulfilling the same purpose as `sendSingleAlert` does in twilio world?

Reply:
answered me own question